gpm_api/utils/geospatial.py

```diff
@@ -50,7 +155,7 @@ def get_extent(xr_obj, padding=0):
     lat_min = max(-90, np.nanmin(lat).item() - padding[1])
     lat_max = min(90, np.nanmax(lat).item() + padding[1])
 
-    if lon_min > lon_max:
+    if lon_min > lon_max:  # TODO: this condition is never met
```

Comment:
We can have a function `_is_crossing_dateline` checking if two consecutive longitude values are separate by more than 360 degree (-180 --> 180) or viceversa ...  and raise error if this occurs. 
Note that for PMW polar orbit satellites, when the swath overpass the pole we also cross the dateline ...